spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -382,6 +382,14 @@ def __init__(self, parent=None):
         self.completion_widget.sig_completion_hint.connect(
             self.show_hint_for_completion)
 
+        # Handle calls to document_did_change when typing text.
+        # See: _handle_keypress_event.
+        self._timer_call_document_did_change = QTimer(self)
+        self._timer_call_document_did_change.setSingleShot(True)
+        self._timer_call_document_did_change.setInterval(250)

```

Comment:
Instead of a magic number embedded in this call, I suggest making it either a class or a module constant and calling it that way.